### PR TITLE
Add Celo token support in SDK

### PR DESCRIPTION
**Pull request by wesgraham**
Adds relevant configs for supporting celo tokens in SDK
Ticket: BG-24044

### CODE REVIEW

File: modules/core/src/config.ts [hunk 1]
```diff
@@ -98,6 +122,9 @@ export const tokens: Tokens = {
     ofc: {
       tokens: formattedOfcCoins.filter(token => coins.get(token.type).network.type === NetworkType.TESTNET),
     },
+    celo: {
+      tokens: formattedCeloTokens.filter(token => token.network === 'Testnet'),
```
Comment (marktoda) [inline modules/core/src/config.ts:126]: NetworkType.TESTNET

File: modules/core/src/config.ts [hunk 2]
```diff
@@ -86,6 +107,9 @@ export const tokens: Tokens = {
     ofc: {
       tokens: formattedOfcCoins.filter(token => coins.get(token.type).network.type === NetworkType.MAINNET),
     },
+    celo: {
+      tokens: formattedCeloTokens.filter(token => token.network === 'Mainnet'),
```
Comment (marktoda) [inline modules/core/src/config.ts:111]: NetworkType.MAINNET

**Comment by wesgraham** [comment]
Will polish the interface for ERC20 like tokens to match the interface we have in platform (`CeloToken -> abstractEthLikeToken -> AbstractEthLikeCoin`) in a future ticket but need support for celo tokens ASAP

File: modules/core/src/v2/coins/celoToken.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  network: string;
+  tokenContractAddress: string;
+  decimalPlaces: number;
+}
+
+export class CeloToken extends Celo {
+  public readonly tokenConfig: CeloTokenConfig;
+
+  constructor(bitgo: BitGo, tokenConfig: CeloTokenConfig) {
+    const staticsCoin = tokenConfig.network === 'Mainnet' ? coins.get('celo') : coins.get('tcelo');
```
Comment (marktoda) [inline modules/core/src/v2/coins/celoToken.ts:23]: NetworkType.MAINNET

**Review by marktoda — CHANGES_REQUESTED** [review]

**Review by marktoda — APPROVED** [review]
oh, maybe the enum comparison won't work on second thought

**Comment by wesgraham** [comment]
> oh, maybe the enum comparison won't work on second thought

Will ignore in that case!

**Pull request merged**